src/components/content/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import classNames from 'classnames';
+import styles from './style.css';
+
+function addClassNames (element) {
+  if (styles[element.type]) {
```

Comment:
What do we think of writing the following code using a guard clause instead of the `if/else` approach?

```
if (!styles[element.type]) {
     return element;
}

return React.cloneElement(element, {
     className: classNames(element.props.className, styles[element.type])
});
```

As per http://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html

Maybe not so relevant here, but more as a general approach.